src/options/Options.tsx

```diff
@@ -220,7 +220,7 @@ export default class Options extends Component<Props, State> {
         <Divider horizontal>Domains</Divider>
         <Form.Checkbox
           toggle
-          label="Cache domain qualities and compare them on the next visit"
+          label="Cache domain qualities to compare them on the next visit"

```

Comment:
Add something regarding the blocking behavior that will be activated and deactivated by toggling this switch.